src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -664,4 +756,20 @@ li {
 ::v-deep .v-input--checkbox .v-messages:not(.error--text) {
   display: none;
 }
+
+::v-deep {
```

Comment:
Please add a comment explaining what you're overriding here.

Reply:
For sure

Reply:
Updated